app/src/androidTest/java/com/github/houseorganizer/houseorganizer/MainScreenActivityTest.java

```diff
@@ -45,6 +50,9 @@ public static void createMockFirebase() throws ExecutionException, InterruptedEx
 
         db = FirebaseFirestore.getInstance();
         auth = FirebaseAuth.getInstance();
+
+        auth.signOut();
+        FirebaseTestsHelper.signInTestUserWithCredentials(FirebaseTestsHelper.TEST_USERS_EMAILS[7], FirebaseTestsHelper.TEST_USERS_PWD[7]);
```

Comment:
I am afraid that signin-in with a particular user before beginning the tests might break them. Maybe only sign-in the user at the start of the tests that need it ?